source/gui/settingsDialogs.py

```diff
@@ -3340,6 +3341,159 @@ def onSave(self):
 		config.conf["addonStore"]["automaticUpdates"] = [x.value for x in AddonsAutomaticUpdate][index]
 
 
+class RemoteSettingsPanel(SettingsPanel):
+	# Translators: This is the label for the remote settings category in NVDA Settings screen.
+	title = _("Remote")
+	autoconnect: wx.CheckBox
+	client_or_server: wx.RadioBox
+	connection_type: wx.RadioBox
+	host: wx.TextCtrl
+	port: wx.SpinCtrl
+	key: wx.TextCtrl
+	play_sounds: wx.CheckBox
+	delete_fingerprints: wx.Button
+
+	def makeSettings(self, settingsSizer):
+		self.config = configuration.get_config()
+		sHelper = gui.guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
+		self.autoconnect = wx.CheckBox(
+			parent=self,
+			id=wx.ID_ANY,
+			# Translators: A checkbox in add-on options dialog to set whether NVDA should automatically connect to a control server on startup.
```

Comment:
I think you would like to remove references to add-on in translator comments, since this is NVDA's core.